extensions/indexes/lucene/src/main/java/org/exist/xquery/modules/lucene/Facets.java

```diff
@@ -42,13 +45,13 @@
                         "to the entire sequence returned by ft:query, so the same map will be returned for all nodes in the sequence. " +
                         "It is thus sufficient to specify one node from the sequence as first argument to this function.",
                 new SequenceType[] {
-                        new FunctionParameterSequenceType("node", Type.NODE, Cardinality.EXACTLY_ONE,
+                        new FunctionParameterSequenceType("node", Type.NODE, Cardinality.ZERO_OR_MORE,
```

Comment:
Description needs to be updated to match the new cardinality. Also is this meant to be `ZERO_OR_MORE` instead of `ONE_OR_MORE`?

Reply:
Yes, indeed. I corrected the description. `ZERO_OR_MORE` is correct. The function will return an empty map if an empty sequence is passed in.